src/routes.json

```diff
@@ -220,6 +148,41 @@
         "title": "Community Moderator"
     },
     {
+        "name": "microworld-art",
+        "pattern": "^/microworlds/art",
+        "routeAlias": "/microworlds",
+        "view": "microworld/art/art",
+        "title": "Art"
+    },
+    {
+        "name": "microworld-fashion",
+        "pattern": "^/microworlds/fashion",
+        "routeAlias": "/microworlds",
+        "view": "microworld/fashion/fashion",
+        "title": "Fashion"
+    },
+    {
+        "name": "microworld-hiphop",
+        "pattern": "^/microworlds/hiphop",
+        "routeAlias": "/microworlds",
+        "view": "microworld/hiphop/hiphop",
+        "title": "Hip Hop Dance"
+    },
+    {
+        "name": "microworld-soccer",
+        "pattern": "^/microworlds/soccer",
+        "routeAlias": "/microworlds",
+        "view": "microworld/soccer/soccer",
+        "title": "Soccer"
+    },
+    {
+        "name": "microworlds-homepage",
```

Comment:
should this go above `microworlds-soccer`? I don't feel strongly about it, just checking!